packages/db/test/pinning.spec.js

```diff
@@ -390,24 +398,33 @@ describe('Pin Request', () => {
     })
 
     it('throws if the auth key does not belong to the pin request', async () => {
-      assert.rejects(client.deletePsaPinRequest(parseInt(aPinRequestOutput._id, 10), 'fakeAuth'))
+      assert.rejects(client.deletePsaPinRequest(aPinRequestOutput._id, 'fakeAuth'))
     })
 
     it('returns the id of the deleted pin request', async () => {
+<<<<<<< HEAD
+      const aPinRequestOutputId = aPinRequestOutput._id
+      const pinRequest = await client.getPsaPinRequest(aPinRequestOutputId)
+=======
       const aPinRequestOutputId = parseInt(aPinRequestOutput._id, 10)
       const pinRequest = await client.getPsaPinRequest(authKey, aPinRequestOutputId)
+>>>>>>> feat/pinning-apis
       assert.ok(!pinRequest.deleted, 'is null')
       const deletedPinRequest = await client.deletePsaPinRequest(aPinRequestOutputId, authKey)
       assert.ok(deletedPinRequest)
       assert.equal(deletedPinRequest._id, pinRequest._id)
     })
 
     it('does not select pin request after deletion', async () => {
+<<<<<<< HEAD
+      assert.rejects(client.getPsaPinRequest(aPinRequestOutput._id))
+=======
       assert.rejects(client.getPsaPinRequest(authKey, parseInt(aPinRequestOutput._id, 10)))
+>>>>>>> feat/pinning-apis
```

Comment:
Some merge conflicts here. Looks like it should be authKey plus id not parsed to int?